models/public/efficientnet-b0-pt/model.yml

```diff
@@ -0,0 +1,73 @@
+# Copyright (c) 2019 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+description: >-
+  The `efficientnet-b0-pt` model is one of the efficientnet <https://arxiv.org/abs/1905.11946>
```

Comment:
You've been using the `-pytorch` suffix for PyTorch models so far, and I think you should stick to it.

Reply:
I want to change `-pytorch` suffix to `-pt` for simplicity and consistency with another frameworks.

Reply:
I would rather you didn't. The model names are effectively part of the downloader API, so they should not be changed unless absolutely necessary.

Reply:
Done